### ISSUE TITLE
Annotation file for multiple objects of single class in an image

### ISSUE THREAD

**Issue by talhaanwarch** [fizyr/keras-retinanet#1345]
If i have multiple objects of same class in an image. Should i create annotation file like this
```
/data/imgs/img_001.jpg,837,346,981,456,cat
/data/imgs/img_001.jpg,215,312,279,391,cat
/data/imgs/img_001.jpg,22,5,89,84,cat
/data/imgs/img_002.jpg,,,,,
```
and if i have single class, i am not supposed to do classification, so i need only regression loss function? Is it?
If yes, how can i pass this as argument in following
`!keras_retinanet/bin/train.py --freeze-backbone --random-transform --weights {PRETRAINED_MODEL}  --batch-size 8  --epochs 2 csv annotations.csv classes.csv`

**Comment by hgaiser** [comment]
> If i have multiple objects of same class in an image. Should i create annotation file like this

Yes

> and if i have single class, i am not supposed to do classification, so i need only regression loss function? Is it?

No, there still needs to be a classification, but it will just be one class.

**Comment by wangcc123-dave** [comment]
What about multiple objects of differents classes in an image? Change  like this?
/data/imgs/img_001.jpg,837,346,981,456,cat
/data/imgs/img_001.jpg,215,312,279,391,cow
/data/imgs/img_001.jpg,22,5,89,84,dog

**Comment by hgaiser** [comment]
Yep (this is all in the README.md by the way ;) )
